README.md

```diff
@@ -67,6 +69,11 @@ const commands = new SlashCommandRegistry()
                 .setHandler(interaction => interaction.reply('User info'))
             )
         )
+    )
+    .addContextMenuCommand(command => command
+        .setName('select')
```

Comment:
It seems that this should have a better name. Like maybe prefix all of these context menu commands with the name of the bot?
![image](https://user-images.githubusercontent.com/24233409/145318086-b3397031-5144-44ed-895e-878adac1fe98.png)

Reply:
This repo isn't the bot itself. This is the command handling backbone, and that particular block of code is just an example in the README. Your point is good anyway though, since I do use "select" as the command name in the bot.